### PR TITLE
Add auto game accept option

### PR DESCRIPTION
**Pull request by zitreF**
Sometimes people go AFK, it finds game and they cant accept, so they have to wait another ~5 minutes, its annoying. This feature will allow players to automate game acceptation to avoid situations like that, but they still will get notified :)

Also some code cleanup and rounded avatar for better look and feel

### CODE REVIEW

File: SwingUI/src/main/java/com/hawolt/ui/queue/QueueWindow.java [hunk 1]
```diff
@@ -213,6 +213,22 @@ public void onEvent(Event event) throws Exception {
             AudioEngine.play(Sound.QUEUE_POP);
             long maxAfkMillis = teamBuilderPopEvent.getMaxAfkMillis();
             this.swiftrift.getLeagueClient().cache(CacheElement.CHAMP_SELECT_COUNTER, teamBuilderPopEvent.getCounter());
+            int queueId = teamBuilderPopEvent.getQueueId();
+            if (queueId == 1090 || queueId == 1100 || queueId == 1130 || queueId == 1160) {
```
Comment (xthesebx) [inline SwingUI/src/main/java/com/hawolt/ui/queue/QueueWindow.java:217]: would change this
```suggestion
            if (currentMode.equals("TFT")) {
```

Comment (xthesebx) [reply SwingUI/src/main/java/com/hawolt/ui/queue/QueueWindow.java:217]: rest looks fine

**Comment by zitreF** [comment]
dont merge it yet, I have last thing to change

**Comment by zitreF** [comment]
should be better now

**Pull request closed without merging**